Bindings/Java/Matlab/Dynamic_Walker_Example/DesignMainStarter_Manager.m

```diff
@@ -0,0 +1,75 @@
+function J = DesignMainStarter_Manager(states)
+% This script opens a Model, edits its initial state, runs a forward 
+% simulation.
+
+display(states)
+
+% Import
+import org.opensim.modeling.*;
+
+% Open a Model by name
+model = Model('../Model/WalkerModel.osim');
+
+% Initialize the system and get the initial state
+s = model.initSystem();
+
+% Set the initial states of the model
+editableCoordSet = model.updCoordinateSet();
+editableCoordSet.get(3).setValue(s, states(1)); % LHIP
+editableCoordSet.get(4).setValue(s, states(1)); % RHIP
+editableCoordSet.get(5).setValue(s, states(1)); % LKnee
+editableCoordSet.get(6).setValue(s, states(1)); % RKnee
+
+% add a reporter
+reporter = TableReporter();
+reporter.set_report_time_interval(0.1)
+
+% Add inputs to the reporter
+reporter.updInput('inputs').connect( model.getCoordinateSet().get(1).getOutput('value'), 'Pelvis_tx')
+reporter.updInput('inputs').connect( model.getCoordinateSet().get(3).getOutput('value'), 'Hip_Angle_L')
+reporter.updInput('inputs').connect( model.getCoordinateSet().get(4).getOutput('value'), 'Hip_Angle_R')
+reporter.updInput('inputs').connect( model.getCoordinateSet().get(5).getOutput('value'), 'Knee_Angle_L')
+reporter.updInput('inputs').connect( model.getCoordinateSet().get(6).getOutput('value'), 'Knee_Angle_R')
+
+% add the reporter to the model
+model.addComponent(reporter) 
+
+% re-initialize the state
+state = model.initSystem();
+
+% Simulate.
+manager = Manager(model);
+manager.setInitialTime(0); 
+manager.setFinalTime(3);
+manager.integrate(s);
+
+% get the table last value of pelvis X (distance down the run way)
+table = reporter.getTable();
+nRows = table.getNumRows();
+
+%display(table.getDependentColumnAtIndex(0).getElt(nRows-1,0))
+
+% need to get close to 3 meters
+J = 3 - table.getDependentColumnAtIndex(0).getElt(nRows-1,0) ;
```

Comment:
If you want "close to 3 meters" as indicated in the comment, you need `abs()` to penalize simulations that exceed 3 m. Perhaps simply set J to the negative of the distance traveled, where lower is better?